src/style.css

```diff
@@ -0,0 +1,55 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.card {
+  width: 96px;
+  height: 96px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  background-color: #fff;
+}
+
+.card__row {
+  display: flex;
+  justify-content: space-between;
+}
+
+.card__item {
+  width: 16px;
+  height: 16px;
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-size: auto;
+  background-position: center;
+  margin: 0 2px;
+}
+
+.card__item:first-child {
+  margin-left: 0;
+}
+
+.card__item:last-child {
+  margin-right: 0;
+}
+
+.stars--1 .card__item:nth-child(1),
+.stars--2 .card__item:nth-child(1),
+.stars--3 .card__item:nth-child(1),
+.stars--4 .card__item:nth-child(1),
+.stars--5 .card__item:nth-child(1),
+.stars--2 .card__item:nth-child(2),
+.stars--3 .card__item:nth-child(2),
+.stars--4 .card__item:nth-child(2),
+.stars--5 .card__item:nth-child(2),
+.stars--3 .card__item:nth-child(3),
+.stars--4 .card__item:nth-child(3),
+.stars--5 .card__item:nth-child(3),
+.stars--4 .card__item:nth-child(4),
+.stars--5 .card__item:nth-child(4),
+.stars--5 .card__item:nth-child(5) {
+  background-image: url(./images/star-active.svg);
+}
```

Comment:
You have to use `nth-child(n)` syntax, read about it in the [documentation](https://developer.mozilla.org/en-US/docs/Web/CSS/:nth-child)